core/common/src/main/java/alluxio/ClientContext.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
+
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.conf.InstancedConfiguration;
+import alluxio.util.ConfigurationUtils;
+
+import javax.annotation.Nullable;
+import javax.security.auth.Subject;
+
+/**
+ * A ClientContext contains information about a security subject and Alluxio configuration which
+ * is used to perform client operations in an Alluxio cluster.
+ */
+public class ClientContext {
+
+  private final AlluxioConfiguration mConf;
+  private final Subject mSubject;
+
+  /**
+   * A client context with information about the subject and configuration of the client.
+   *
+   * @param subject The security subject to use
+   * @param alluxioConf The {@link AlluxioConfiguration} to use. If null, the site property defaults
+   * will be loaded
```

Comment:
aligned or indented by 2 spaces? 4 spaces?

Reply:
4 (according to google's style guide)

Reply:
They say 4 or more - the way we indent in the rest of our codebase is so that the indentation lines up with the variable name.
```java
/**
 * @param abc ....................................................
 *        ......................
 */
```